### PR TITLE
Begin to create mobile version of app

### PR DESCRIPTION
**Pull request by krogowsk531**
# Katy

- [x] checked functionality of the webpage
- [ ] received help from a mentor/collaborator?
- [x] added teammates as reviewers

## Notes
* Began work on creating mobile version of app
* About halfway through completing all accessibility in HTML
* Had to add new images
* Created a new file that will be refactored for SCSS to hold media queries (media-queries.scss)

## Issues Addressed
* Closes #<ticket number>
* Addresses #48 
* Addresses #18

## Questions
*

## Problems or Challenges
*

## Screenshots (if applicable)
*

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
     <link rel="stylesheet" type="text/scss" href="./css/index.scss">
     <link href="https://fonts.googleapis.com/css?family=Quicksand:400,600,700&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Sulphur+Point:400,700&display=swap" rel="stylesheet">
-    <link rel="icon" href="./images/apple-logo.png" />
-    <link rel="icon" href="./images/search.png" />
-    <link rel="icon" href='./images/cookbook.png'/>;
-    <link rel="icon" href='./images/seasoning.png'/>;
-    <link rel="icon" href="./images/apple-logo-outline.png"/>
-    <link rel="icon" href="./images/shopping-cart-icon.png"/>
+    <link rel="icon" href="./images/apple-logo.png" alt="image of an apple"/>
```
Comment (GreyMatteOr) [inline src/index.html:10]: Thanks for adding all these meta tags on all the images and such! Very helpful

File: src/css/media-queries.scss [hunk 2]
```diff
@@ -0,0 +1,147 @@
+// @mixin iphone-size {
```
Comment (GreyMatteOr) [inline src/css/media-queries.scss:1]: Nice! I hadn't seen putting media queries in a separate file before. Makes a lot of sense, definitely stealing this!

**Review by kathrynljackson — APPROVED** [review]
I love that you separated the mobile view scss into its own directory!

**Review by GreyMatteOr — APPROVED** [review]
Learning a lot. Really appreciate all the smart Ideas you implemented

**Pull request merged**